### ISSUE TITLE
`dimension-no-non-numeric-values` throws error

### ISSUE THREAD

**Issue by fuhlig** [kristerkari/stylelint-scss#363]
Using the [demension-no-non-numeric-values](https://github.com/kristerkari/stylelint-scss/tree/master/src/rules/dimension-no-non-numeric-values) introduced in 3.10 I get the following error

```
// config
'scss/dimension-no-non-numeric-values': true

// stylesheet
$value: 4;

p {
  padding: #{value}px;
}
```

```
TypeError: message.text.replace is not a function
    at orderedMessages.map.message (\node_modules\stylelint\lib\formatters\stringFormatter.js:124:10)
    at Array.map (<anonymous>)
    at formatter (\node_modules\stylelint\lib\formatters\stringFormatter.js:113:43)
    at results.reduce (\node_modules\stylelint\lib\formatters\stringFormatter.js:182:15)
    at Array.reduce (<anonymous>)
    at module.exports (\node_modules\stylelint\lib\formatters\stringFormatter.js:168:20)
    at prepareReturnValue (\node_modules\stylelint\lib\standalone.js:285:15)
    at globby.then.then.stylelintResults (\node_modules\stylelint\lib\standalone.js:274:14)
```

The code inside the stylesheet does not matter. Setting the rule to `true` causes the error.

**Comment by kristerkari** [comment]
Thanks, I'll have a look!

**Comment by kristerkari** [comment]
There is a pull request open with a fix: #365
